src/shared/gpt.h

```diff
@@ -25,13 +45,7 @@ typedef struct GptPartitionType {
         sd_id128_t uuid;
         const char *name;
         Architecture arch;
-
-        bool is_root:1;
-        bool is_root_verity:1;
-        bool is_root_verity_sig:1;
-        bool is_usr:1;
-        bool is_usr_verity:1;
-        bool is_usr_verity_sig:1;
+        GptPartitionIdentifier id;
```

Comment:
`id` is a bit of a misnomer, this is not about identifcatin of the partition type but really about designating a purpose for this partition, hence "designator" or so, as with `PartitionDesignator`, see above